hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -2857,8 +2863,15 @@ private void validate() {
     }
 
     public HoodieWriteConfig build() {
+      return build(true);
+    }
+
+    @VisibleForTesting
+    public HoodieWriteConfig build(boolean shouldValidate) {
```

Comment:
should this method use the default scope/visibility instead of `public`?

Reply:
Problem is that it's being used in tests in different packages

Reply:
OK.  We should think about how to hide this, so it is not used by users accidentally.